commitlintplugins.test.ts

```diff
@@ -348,11 +348,36 @@ test('proper-issue-refs3', () => {
     let commitMsgWithHashtagRefInBlock =
         "foo: this is only a title" + "\n\n" + "Bar baz:\n\n```\ntype Foo = string #123\n```";
     let properIssueRefs3 = runCommitLintOnMsg(commitMsgWithHashtagRefInBlock);
-    console.log('HERE =======>' + properIssueRefs3.stdout)
     expect(properIssueRefs3.status).toBe(0);
 });
 
 
+test('proper-revert-message1', () => {
+    let commitMsgWithoutProperRevertMessage = 
+        'Revert "add abbreviations.ts"\n\n' + 
+        'This reverts commit 0272f587c7eece147e8d1756116b0b43e11c34ac.';
+    let properRevertMessage1 = runCommitLintOnMsg(commitMsgWithoutProperRevertMessage);
+    expect(properRevertMessage1.status).not.toBe(0);
+});
+
+
+test('proper-revert-message2', () => {
+    let commitMsgWithProperRevertMessage = 
+        'Revert "add abbreviations.ts"\n\n' + 
+        'This reverts commit 0272f587c7eece147e8d1756116b0b43e11c34ac\n' +
+        'because/otherwise bla bla.'
```

Comment:
@realmarv let's add a test4 that explains the reason in same line where the text "This reverts commit" is

Reply:
That line's length is already 60 characters. If I do that, the test will fail because of the body-max-length rule. unless I say "This reverts commit bellow because bla bla" and put the commit hash in the next line. What do you think?

Reply:
I knew you would say that, but I said this because I think your logic is wrong, because it's only looking at the first line. And you're doing that because you know our limit is 64, but what if we increase the limit in the future? adding the reason in the first line could fit in this case, and your rule would have a bug.

Reply:
Right. So should I find a way to set a higher value for body max length in that test? (Because now the limit is 64 and it will fail anyway) Also, what do you think about my suggestion:
> I say "This reverts commit bellow because bla bla" in the first line of the body and put the commit hash in the next line.

Reply:
I just want a commit message that includes the This reverts commit xyz. in the 2nd line, but with more text in the 2nd line. Because this is a testcase that the rule should not complain about, because the user has modified the default revert message.

Reply:
is this resolved?

Reply:
Yes